src/Materials/babylon.materialHelper.ts

```diff
@@ -183,6 +183,10 @@ module BABYLON {
                         defines["BonesPerMesh"] = (mesh.skeleton.bones.length + 1);
                         defines["BONETEXTURE"] = materialSupportsBoneTexture ? false : undefined;
                     }
+
+                    if (mesh.skeleton.ignoreMeshTransform) {

```

Comment:
Should be defines["IGNORETRANSFORM"] = mesh.skeleton.ignoreMeshTransform to support changing the value dynamically

Reply:
yes, that's a good point.

Reply:
no longer making changes to shader